bin/get-video-s3-acls.py

```diff
@@ -0,0 +1,409 @@
+                "",  # gloss_public,
+                "",  # video_public,
+            ]
+
+    # Find NZSL keys that are absent from S3 (present handled above)
+    for video_key, [
+        gloss_idgloss,
+        gloss_created_at,
+        gloss_id,
+        video_id,
+        gloss_public,
+        video_public,
+    ] in this_nzsl_raw_keys_dict.items():
+        if video_key not in this_s3_bucket_raw_keys_list:
+            this_all_keys_dict[video_key] = [
+                True,   # NZSL PRESENT
+                False,  # S3 Absent
+                gloss_idgloss,
+                gloss_created_at,
+                gloss_id,
+                video_id,
+                gloss_public,
+                video_public,
+            ]
+
+    return this_all_keys_dict
+
+
+def build_csv_header():
+    return CSV_DELIMITER.join(
+        [
+            "S3 Video key",
+            "Sbank Gloss",
+            "Sbank Gloss created at",
+            "S3 LastModified",
+            "S3 Expected Canned ACL",
+            "S3 Actual Canned ACL",
+            "Sbank Gloss ID",
+            "Sbank Video ID",
+            "Sbank Gloss public",
+            "Sbank Video public",
+            "Action",
+        ]
+    )
+
+
+def build_csv_row(
+    key_in_nzsl=False,
+    key_in_s3=False,
+    video_key=None,
+    gloss_idgloss=None,
+    gloss_created_at=None,
+    gloss_id=None,
+    video_id=None,
+    gloss_public=False,
+    video_public=False,
+):
+
+    action = ""
+    # Cases
+    # In S3     In NZSL     Action
+    #   Is        No          Delete!
+    #   Is        Is          Check ACL
+    #   Not       Is          Review
+    #   (F F impossible)
+
+    if key_in_s3:
+        if key_in_nzsl:
+            action = "Check ACL"
+        else:
+            action = "Delete"
+    else:
+        action = "Review"
+
+    # See signbank/video/models.py, line 59, function set_public_acl()
+    if key_in_nzsl:
+        canned_acl_expected = "public-read" if video_public else "private"
+    else:
+        canned_acl_expected = ""
+
+    # If key not in S3, just return its NZSL info and action
+    if not key_in_s3:
+        return CSV_DELIMITER.join(
+            [
+                f"{video_key}",
+                f"{gloss_idgloss}",
+                f"{gloss_created_at}",
+                "",  # S3 LastModified
+                f"{canned_acl_expected}",
+                "",  # Actual Canned ACL
+                f"{gloss_id}",
+                f"{video_id}",
+                f"{gloss_public}",
+                f"{video_public}",
+                action,
+            ]
+        )
+
+    # Get S3 object's ACL
+    result = subprocess.run(
+        [
+            AWSCLI,
+            "s3api",
+            "get-object-acl",
+            "--output",
+            "json",
+            "--bucket",
+            AWS_S3_BUCKET,
+            "--key",
+            video_key,
+        ],
+        env=os.environ,
+        shell=False,
+        check=True,
```

Comment:
suggestion: this is the default value for the param:

```suggestion
        env=os.environ,
        check=True,
```

Reply:
Done.